[2022-08-23 22:46:01,349] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: gmailPipelineV1.gmailExtractTask scheduled__2022-08-23T00:00:00+00:00 [queued]>
[2022-08-23 22:46:01,354] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: gmailPipelineV1.gmailExtractTask scheduled__2022-08-23T00:00:00+00:00 [queued]>
[2022-08-23 22:46:01,354] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 22:46:01,354] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-23 22:46:01,354] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 22:46:01,363] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): gmailExtractTask> on 2022-08-23 00:00:00+00:00
[2022-08-23 22:46:01,365] {standard_task_runner.py:52} INFO - Started process 11446 to run task
[2022-08-23 22:46:01,370] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'gmailPipelineV1', 'gmailExtractTask', 'scheduled__2022-08-23T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/gmailDag.py', '--cfg-path', '/var/folders/mm/xv24gcjj46l_jx1m76p5bgy80000gn/T/tmp_ul976vc', '--error-file', '/var/folders/mm/xv24gcjj46l_jx1m76p5bgy80000gn/T/tmpa1rrlrmi']
[2022-08-23 22:46:01,372] {standard_task_runner.py:80} INFO - Job 24: Subtask gmailExtractTask
[2022-08-23 22:46:01,417] {task_command.py:371} INFO - Running <TaskInstance: gmailPipelineV1.gmailExtractTask scheduled__2022-08-23T00:00:00+00:00 [running]> on host abhays-mbp.lan
[2022-08-23 22:46:01,467] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=gmailPipelineV1
AIRFLOW_CTX_TASK_ID=gmailExtractTask
AIRFLOW_CTX_EXECUTION_DATE=2022-08-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-23T00:00:00+00:00
[2022-08-23 22:46:01,478] {__init__.py:49} INFO - file_cache is only supported with oauth2client<4.0.0
[2022-08-23 22:46:01,946] {logging_mixin.py:115} INFO - No emails to read!!
Attention empty msglist is being passed
[2022-08-23 22:46:01,948] {__init__.py:49} INFO - file_cache is only supported with oauth2client<4.0.0
[2022-08-23 22:46:02,352] {logging_mixin.py:115} INFO - An error occurred: <HttpError 400 when requesting https://gmail.googleapis.com/gmail/v1/users/me/messages/batchModify? returned "No message ids specified.". Details: "[{'message': 'No message ids specified.', 'domain': 'global', 'reason': 'invalidArgument'}]">
[2022-08-23 22:46:02,353] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 22:46:02,361] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=gmailPipelineV1, task_id=gmailExtractTask, execution_date=20220823T000000, start_date=20220824T024601, end_date=20220824T024602
[2022-08-23 22:46:02,384] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 22:46:02,405] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
